File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/metaschema/Soglia.ts
import { Lavorazione } from "../attivita/Lavorazione";
import { BaseCodeDescr } from "../baseClass/baseCodeDescr";
import { Avversita } from "./avversita/Avversita";
import { UnitaDiMisura } from "./UnitaDiMisura";

export class Soglia extends BaseCodeDescr {

    quantita: number;
    avversita: Avversita;
    udm: UnitaDiMisura;
    lavorazione: Lavorazione;

    constructor(codice: number) {
        super(codice);
    }

}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/metaschema/Soglia.cs
using AgronicaCoreModelsSTD.baseClass;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.metaschema
{
    public class Soglia : BaseCodeDescr
    {
        public decimal quantita { get; set; }

        public avversita.Avversita avversita { get; set; }
        public UnitaDiMisura udm { get; set; }
        public attivita.Lavorazione lavorazione { get; set; }


        public Soglia(int codice) : base(codice, "")
        {
        }
        public Soglia() : base() { }
    }
}

